interface/src/app/node/high_weight_genes.spec.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * Test <high-weight-genes> directive.
+ **/
+
+describe('<high-weight-genes> directive', function() {
+  beforeEach(module('adage.node'));
+
+  // Thanks to "grunt-html2js" module (called by Grunt), the template file
+  // "node/high_weight_genes.tpl.html" has been converted into an Angular
+  // module of the same name and put in $templateCache before karma runs.
+  beforeEach(module('node/high_weight_genes.tpl.html'));
```

Comment:
Line 11 actually can be merged with line 6 into the following line:
```
beforeEach(module('adage.node', 'node/high_weight_genes.tpl.html'));
```
I am writing them separately to emphasize that the HTML template file **node/high_weight_genes.tpl.html** has been converted into a JavaScript module by **grunt-html2js**.